### ISSUE TITLE
submission script prep documentation

### ISSUE THREAD

**Issue by raynamharris** [nih-cfde/published-documentation#358]
The first step in the Usage section of the [submission script prep wiki](https://github.com/nih-cfde/published-documentation/wiki/submission-prep-script) reads like this:

"0. First build your file.tsv, biosample.tsv, biosample_disease.tsv, biosample_gene.tsv, biosample_substance.tsv, subject_disease.tsv, subject_phenotype.tsv, subject_role_taxonomy.tsv, subject_substance.tsv, and collection_disease.tsv tables. (Some of these can be left empty (as header-only TSVs) if desired: see the [C2M2 table wiki](https://github.com/nih-cfde/published-documentation/wiki/C2M2-Table-Summary) for requirements.)"

I find this first step to be a little confusing because all of the files listed _can_ be left blank while none of the files that _cannot_ be left blank are mentioned. Also, there is no link to the blank tsv files. To make what files can and cannot be left blank a little more explicit, can we reword this section to something like:

0. Prepare the required [id_namespace.tsv](https://github.com/nih-cfde/published-documentation/wiki/TableInfo:-id_namespace.tsv), [project.tsv](https://github.com/nih-cfde/published-documentation/wiki/TableInfo:-project.tsv), and [dcc.tsv](https://github.com/nih-cfde/published-documentation/wiki/TableInfo:-dcc.tsv) files. 
1. Then, prepare the tsv tables for associating meta data with files, biosamples, subjects, etc. tsv files can be left blank if there is no relevant metadata. [Download blank tsv files here](https://osf.io/4q69a/). See the [C2M2 Table Summary](https://github.com/nih-cfde/published-documentation/wiki/C2M2-Table-Summary) for requirements. (Make sure all id_namespaces in the tsv files are represented in the [id_namespace.tsv](https://github.com/nih-cfde/published-documentation/wiki/TableInfo:-id_namespace.tsv) mentioned above.)

**Comment by abradyIGS** [comment]
Agreed on clarity/confusion: I think the whole page could benefit from reconstitution. (By generous arrangement with @raynamharris) I'll draft a top-down rewrite within 24 hours and re-present for further evaluation.
